lib/rules/space-before-function-paren.js

```diff
@@ -133,7 +158,8 @@ module.exports = {
 
         return {
             FunctionDeclaration: validateSpacingBeforeParentheses,
-            FunctionExpression: validateSpacingBeforeParentheses
+            FunctionExpression: validateSpacingBeforeParentheses,
+            ArrowFunctionExpression: validateSpacingBeforeParentheses,
```

Comment:
I don't see why we should. Most rules don't go for alphabetical ordering. Also, I see FunctionExpression and ArrowFunctionExpression as being natural siblings here (FunctionDeclaration is statement position only). So to me this is a natural ordering.